pkg/hostagent/requirements.go

```diff
@@ -104,15 +104,15 @@ A possible workaround is to run "apt-get install sshfs" in the guest.
 `,
 		})
 		req = append(req, requirement{
-			description: "/etc/fuse.conf to contain \"user_allow_other\"",
+			description: "/etc/fuse(3).conf to contain \"user_allow_other\"",
 			script: `#!/bin/bash
 set -eux -o pipefail
-if ! timeout 30s bash -c "until grep -q ^user_allow_other /etc/fuse.conf; do sleep 3; done"; then
-	echo >&2 "/etc/fuse.conf is not updated to contain \"user_allow_other\""
+if ! timeout 30s bash -c "until grep -q ^user_allow_other /etc/fuse.conf || grep -q ^user_allow_other /etc/fuse3.conf; do sleep 3; done"; then
```

Comment:
Can be written as:

```suggestion
if ! timeout 30s bash -c "until grep -q ^user_allow_other /etc/fuse.conf /etc/fuse3.conf; do sleep 3; done"; then
```

I was concerned about what happens when `/etc/fuse3.conf` doesn't exist, but it turns out that `grep -q` will silence that error:

```console
lima-alpine:~# grep ^user_allow_other /etc/fuse.conf /etc/fuse3.conf; echo $?
/etc/fuse.conf:user_allow_other
grep: /etc/fuse3.conf: No such file or directory
2
lima-alpine:~# grep -q ^user_allow_other /etc/fuse.conf /etc/fuse3.conf; echo $?
0
lima-alpine:~# grep -q ^not_found /etc/fuse.conf; echo $?
1
```

This feels a little too magical, so I would also be fine with this:

```suggestion
if ! timeout 30s bash -c "until grep -q ^user_allow_other /etc/fuse*.conf; do sleep 3; done"; then
```

Reply:
Thanks, updated